### PR TITLE
Feature/PJ_SIM_SUNBNB_2023-10-[FE] Review Component

### PR DESCRIPTION
**Pull request by RonnieRII**
# Issue Link
https://framgiaph.backlog.com/view/PJ_SIM_SUNBNB_2023-10

# Definition of done
* Create the review component for users who have checked in with the components:
    * [x] Review Text Field
    * [x] Rating Input Field
    * [x] Submit Button

* Create the review component for users who have checked in with the components:
    * [x] Review from the guest, number of reviews, and an average rating
    * [x] Guest picture and name
    * [x] Rate of Guest
    * [x] Date of Review
    * [x] Comment of Guest
    
# Note
Created three separate pages: one for the review form and two for the reviews on the room page and the hosts' user profile

# Screenshots/Recordings
image
image
image

### CODE REVIEW

**Review by andreenriquez11 — CHANGES_REQUESTED** [review]
Review components should be in components folder.

**Comment by LennardUy104** [comment]
Please edit PR to develop not master. thanks!

**Comment by RonnieRII** [comment]
> Review components should be in components folder.

Created the components folder and moved components into such folder. 

image

**Comment by RonnieRII** [comment]
> Please edit PR to develop not master. thanks!

Changed base from master to develop.

**Review by andreenriquez11 — COMMENTED** [review]
looks good to me :)

**Review by andreenriquez11 — APPROVED** [review]

**Comment by roseaugusto** [comment]
@RonnieRII Please use factories/utilities folder instead of asset, and put the json file inside a dummy folder since this won't be the only one having dummy data.

File: fe/src/router/index.js [hunk 1]
```diff
@@ -15,4 +20,4 @@ const router = createRouter({
   routes,
 });
 
-export default router;
+export default router;
```
Comment (ejanton-sun) [inline fe/src/router/index.js]: add new line below

File: fe/src/utilities/dummy/Review.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[
+    {"id": 1, "rating": 4, "name": "Hiro", "date": "15-JUL-2019", "review": "Very cool host", "img": "https://cdn.britannica.com/71/234471-050-093F4211/shiba-inu-dog-in-the-snow.jpg"},                
+    {"id": 2, "rating": 5, "name": "Maki", "date": "20-AUG-2019", "review": "I enjoyed my stay!", "img": "https://pbs.twimg.com/media/C8QqGm4UQAAUiET.jpg"}
+]
```
Comment (ejanton-sun) [inline fe/src/utilities/dummy/Review.json]: add new line below

File: fe/src/pages/RegistrationView.vue [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 100vh;
+    }
+    .form-content {
+        max-width: 400px;
+        width: 100%;
+    }
+</style>
```
Comment (ejanton-sun) [inline fe/src/pages/RegistrationView.vue]: add new line below

File: fe/src/components/Review/ReviewHost.vue [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+        computed: {
+            averageStarRating() {
+                let sum = 0;
+                revs.forEach((review) => sum += review.rating);
+                return sum / this.reviews.length;
+            }
+        }
+    }
+</script>
```
Comment (ejanton-sun) [inline fe/src/components/Review/ReviewHost.vue]: add new line below

**Review by ejanton-sun — CHANGES_REQUESTED** [review]
Add new line for each EOF

File: fe/src/router/index.js (hunk 1 above)
Comment (RonnieRII) [reply fe/src/router/index.js]: Just added the new line at the end 🙇

File: fe/src/utilities/dummy/Review.json (hunk 2 above)
Comment (RonnieRII) [reply fe/src/utilities/dummy/Review.json]: Just added the new line at the end 🙇

File: fe/src/pages/RegistrationView.vue (hunk 3 above)
Comment (RonnieRII) [reply fe/src/pages/RegistrationView.vue]: Deleted this view as it was not important to the component 🙇

File: fe/src/components/Review/ReviewHost.vue (hunk 4 above)
Comment (RonnieRII) [reply fe/src/components/Review/ReviewHost.vue]: Just added the new line at the end 🙇

**Pull request merged**